src/renderer/families/tron/Votes/index.js

```diff
@@ -52,6 +52,17 @@ const useTronSuperRepresentatives = () => {
   return sp;
 };
 
+// @TODO move this to common
```

Comment:
can we try to keep live-common up to date each time we have these new functions? could be used on mobile

Reply:
i'll make a PR with this :+1: